core/templates/core/Cart.html

```diff
@@ -0,0 +1,241 @@
+                    </p>
+                  </div>
+                  <!-- Data -->
+                </div>
+
+                <div class="col-lg-4 col-md-6 mb-4 mb-lg-0 text-center font-weight">
+                  <!-- Quantity -->
+                  <button type="button" class="btn btn-danger btn-sm me-1 mb-2" data-mdb-toggle="tooltip"
+                    title="Remove product" onclick="removeFromCart('{{ product.product.id }}')">
+                    <a class="fa fa-trash"></a>
+                  </button>
+                  <a type="button" class="btn btn-primary btn-sm mb-2" data-mdb-toggle="tooltip" title="Edit product"
+                    href="{% url 'core:product-detail-view' pk=product.product.id %}">
+                    <i class="fa fa-edit"></i>
+                  </a>
+                </div>
+              </div>
+            </div>
+          </div>
+          {% endfor %}
+          <!--  ------------------------------------------------------------------------------ -->
+        </div>
+      </div>
+      <div class="col-md-4">
+        <div class="card mb-4">
+          <div class="card-header py-3">
+            <h5 class="mb-0">Summary</h5>
+          </div>
+          <div class="card-body">
+            <ul class="list-group list-group-flush">
+              <li class="list-group-item d-flex justify-content-between align-items-center border-0 px-0 pb-0">
+                Products
+                <span id="total-price"></span>
+              </li>
+              <li class="list-group-item d-flex justify-content-between align-items-center px-0">
+                Shipping
+                <span>Gratis</span>
+              </li>
+              <li class="list-group-item d-flex justify-content-between align-items-center border-0 px-0 mb-3">
+                <div>
+                  <strong>Total amount</strong>
+                  <strong>
+                    <p class="mb-0">(including VAT)</p>
+                  </strong>
+                </div>
+                <span><strong id="price-with-tax">$53.98</strong></span>
+              </li>
+            </ul>
+            <button type="button" class="btn btn-primary btn-lg btn-block" id="place-order">
+              Place your order
+            </button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+{% endblock %}
+
+<!-- Footer -->
+{% block footer %}
+<footer>
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <div class="inner-content">
+          <p>Copyright &copy; 2020.
+        
+        - Design: <a rel="nofollow noopener" href="#" target="_blank">Betatesting</a></p>
+        </div>
+      </div>
+    </div>
+  </div>
+</footer>
+{% endblock %}
+
+{% block javascript_tag %}
+
+<script>
+  // Get total price of products
+  function totalPrice() {
+    var price = 0
+    var elems = document.getElementsByName("price");
+    for (var i = 0; i < elems.length; i++) {
```

Comment:
Code performance optimization: 
```suggestion
    const elemsLength = elems.length;
    for (var i = 0; i < elemsLength; i++) {
```
This way length will not be called/used again and again (on every iteration)

Reply:
Great!